### ISSUE TITLE
Grid: Double-click on top row doesn't work when header is not visible

### ISSUE THREAD

**Issue by MrTimbones** [EclipseNebula/nebula#487]
It's not possible to double-click on the first row of a [Grid](https://github.com/eclipse/nebula/blob/master/widgets/grid/org.eclipse.nebula.widgets.grid/src/org/eclipse/nebula/widgets/grid/Grid.java) if the header has not been made visible via `.setHeaderVisible(true)`. The MouseDoubleClick event is simply ignored, and no other listeners are notified.

It looks like this was introduced by PR #175, with this code in `.onMouseDoubleClick()` around [line 6748](https://github.com/eclipse/nebula/blob/7346679047435de18061f20974a633e7fbaeedaf/widgets/grid/org.eclipse.nebula.widgets.grid/src/org/eclipse/nebula/widgets/grid/Grid.java#L6748):
```java
		if (e.y<headerHeight) {
			e.doit = false;
			return;
		}
```
I'm guessing that adding ` && columnHeadersVisible` would probably fix this?

**Comment by lcaron** [comment]
Bug fixed in Nebula 2.7.2

**Comment by lcaron** [comment]
Nebula 2.7.2 is now available
